tests/test_fx/test_meta_info_prop.py

```diff
@@ -13,7 +13,6 @@
 def meta_check(meta_info_spec: TensorMetadata, orig_tensor: torch.Tensor):
     assert meta_info_spec.shape == orig_tensor.shape
     assert meta_info_spec.dtype == orig_tensor.dtype
-    assert meta_info_spec.requires_grad == orig_tensor.requires_grad
```

Comment:
I removed this because backward in MetaInfoProp can't ensure the gradient is correct.
I detached tensors from the autograd graph within each node.